pkg/terror/error_list.go

```diff
@@ -549,512 +549,512 @@ const (
 // Error instances
 var (
 	// Database operation related error
-	ErrDBDriverError = New(codeDBDriverError, ClassDatabase, ScopeNotSet, LevelHigh, "database driver error")
-	ErrDBBadConn     = New(codeDBBadConn, ClassDatabase, ScopeNotSet, LevelHigh, "database driver")
-	ErrDBInvalidConn = New(codeDBInvalidConn, ClassDatabase, ScopeNotSet, LevelHigh, "database driver")
+	ErrDBDriverError = New(codeDBDriverError, ClassDatabase, ScopeNotSet, LevelHigh, "database driver error", "")
```

Comment:
Can we cherry-pick this PR to `release-1.0`? because we may need to copy these error list to `release-1.0` periodically so that every error code has the same meaning in all DM versions?